templates/taxi/driver_list.html

```diff
@@ -5,10 +5,17 @@
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+        {% if driver.id == user.id %}  
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }}) (Me)
+          </li>
+        {% else %}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
The URL tag uses double quotes for the `taxi:driver-detail` name. It's a good practice to use single quotes consistently for URL names to avoid syntax errors.